Resources/public/css/views/universaldiscovery/search.css

```diff
@@ -128,3 +129,12 @@
 .ez-view-universaldiscoverysearchview .ez-searchresult-navigation-link[rel="prev"] {
     padding-right: 1em;
 }
+
+.ez-view-universaldiscoverysearchview .ez-searchresult-content {
+    overflow: auto;
+    height: calc(100% - 6em);
```

Comment:
actually, after digging a bit into the code, maybe there's a way to avoid those magic values and make the whole thing more robust against changes. I had similar issue with the Tree in the browse tab, you can do pretty much the same as https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/css/views/universaldiscovery/browse.css#L48

so basically here given the `ez-ud-pane-search-result-list` element (the parent of `ez-searchresult-content`) has `display: flex`, you can write something like:

```css
.ez-view-universaldiscoverysearchview .ez-searchresult-content {
    height: 100px;
    overflow: auto;
    flex: 1 0 auto;
}
```

as written in the linked comment, the `height` is just there to set a (initial) height on the element so that overflow has an effect, but the actual size will be computed based on the `flex` rule.
It seems to work for me both in Chrome and Firefox and has the benefit of not relying on any other pre-defined size elsewhere.